### ISSUE TITLE
pull or download with --insecure still gives Fatal error: Certificate verification 

### ISSUE THREAD

**Issue by laurenskling** [git-ftp/git-ftp#346]
I can `init` and `push` without problems. Trying to `pull` (because this would be awesome!) will give
```
cd: Fatal error: Certificate verification: self signed certificate (...)
mirror: Fatal error: Certificate verification: self signed certificate (...)
1 error detected
```
According to the documentation `--insecure` should skip this, but as you see it doesn't

my command:
` git-ftp pull --no-commit --insecure -u <myuser> -P --remote-root <myfolder> <myserver>`

**Comment by mkllnk** [comment]
Yes, you are right. The insecure option is not passed to lftp. Lftp supports this via the `ssl:verify-certificate` option. So it should be easy to fix, but I don't have time to do it myself at the moment.

**Comment by laurenskling** [comment]
I don't know anything about lftp, or ftp in general, otherwise I would have loved to help. Is there a way for me to pass commands to lftp from git-ftp?

**Comment by mkllnk** [comment]
Only by modifying git-ftp yourself. It's not too hard. Just open the git-ftp file, it's a shell script. Search for lftp and you will find a command to download everything. You can add the ssl option above to the `lftp_command` variable. Just add `set ssl:verify-certificate no && ` to the beginning of that string.

**Comment by laurenskling** [comment]
Adding this command in the git-ftp file worked :) 
Although `--remote-root` and `--no-commit` aren't respected. 

(my 2 cents, I would never want an automatic merge, i'd like to see the changes first)

**Comment by DerTanzschuh** [comment]
@laurenskling We have finally fixed the issue with the option `--insecure` in #526.

>Although `--remote-root` and `--no-commit` aren't respected.
But I can not confirm this. Both options are working for me as expected.

```sh
$ git ftp pull -v --remote-root html2/
```
This downloads all the files from within html2 directory on the remote (host, user and password are configured in git config). So this is exactly what I wanted. It also works if set as a default with `git config git-ftp.remote-root html2/`.

```sh
$ git ftp pull -v --no-commit
```
And this is working as well. Oh wait … there is more into this … i will create a new issue.

**Comment by DerTanzschuh** [comment]
@laurenskling The issue with `--no-commit` is fixed by #530. If you still have a problem with `--remote-root` I’d recommend to open a extra issue for this so we can discuss it in depth.

**Comment by laurenskling** [comment]
Thanks for the effort guys :) 
I'm not actively using this anymore, but i'm pretty sure you'll make some other guys happy with this <3

**Comment by LukasFritzeDev** [comment]
As said before, this is fixed by #526, so I’ll close  this issue. 

If someone still has a similar problem, feel free to comment here, then we can reopen the issue.
